src/styles/map/_map.scss

```diff
@@ -56,6 +56,24 @@
       margin-bottom: 3px;
     }
   }
+
+  #mapbox-wordmark {
+    position: absolute;
+    display: block;
+    height: 20px;
+    width: 65px;
+    right: 300px;
+    bottom: -1px;
+    text-indent: -9999px;
+    z-index: 99999;
```

Comment:
Hey Nate, can you create a sass variable for this z-index in `styles/_variables.scss`, and then use the var instead of hard-coding? That way we keep all the z-indexes coordinated with each other.

Reply:
done!